src/PKSim.Core/Commands/SetRelativeExpressionFromNormalizedCommand.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using OSPSuite.Core.Domain;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Commands
+{
+   public class SetRelativeExpressionFromNormalizedCommand : PKSimMacroCommand
+   {
+      private readonly double _newNormalizedValue;
+      private readonly IParameter _normalizedParameter;
+
+      public SetRelativeExpressionFromNormalizedCommand(IParameter normalizedParameter, double normalizedValue)
+      {
+         _normalizedParameter = normalizedParameter;
+         _newNormalizedValue = normalizedValue;
+      }
+
+      public override void Execute(IExecutionContext context)
+      {
+         var relativeExpressionParameter = getRelativeExpressionFrom(_normalizedParameter);
+
+         if (relativeExpressionParameter == null)
+            return;
+
+         var relativeExpressionValue = getRelativeExpressionValue(_normalizedParameter, _newNormalizedValue, relativeExpressionParameter);
+
+         Add(new SetRelativeExpressionCommand(relativeExpressionParameter, relativeExpressionValue));
+         Add(new SetParameterValueCommand(_normalizedParameter, _newNormalizedValue));
+
+         base.Execute(context);
+      }
+
+      private double getRelativeExpressionValue(IParameter normalizedParameter, double newNormalizedValue, IParameter relativeExpressionParameter)
+      {
+         return relativeExpressionParameter.Value * newNormalizedValue / normalizedParameter.Value;
+      }
+
+      private IParameter getRelativeExpressionFrom(IParameter normalizedParameter)
+      {
+         return normalizedParameter.ParentContainer.GetAllChildren<IParameter>().FirstOrDefault(isNonNormalizedRelativeExpression);
```

Comment:
This is a little bit confusing here
The non norm parameter is the parameter in the SAME Container having the same name without the NormSuffix
so sthg like
```
name = normalizedParameter.Name.Replace(NormSuffix, "");
return normalizedParameter.ParentContainer.Parameter(name)
```